### PR TITLE
Bump dskit from 528d8a7 to 1725bcc (#9184)

### PR DESCRIPTION
**Pull request by periklis**
**What this PR does / why we need it**:
Manually backporting #9184 into `release-2.8.x`

**Which issue(s) this PR fixes**:
Fixes #<issue number>

**Special notes for your reviewer**:

(cherry picked from commit 3abd26097587779b461a8befef1af06700d0ae13)

**Checklist**
- [x] Reviewed the [`CONTRIBUTING.md`](https://github.com/grafana/loki/blob/main/CONTRIBUTING.md) guide (**required**)
- [x] Documentation added
- [x] Tests updated
- [x] `CHANGELOG.md` updated
- [ ] Changes that require user attention or interaction to upgrade are documented in `docs/sources/upgrading/_index.md`

### CODE REVIEW

**Review by JStickler — APPROVED** [review]
[docs team] LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,29 @@
...
+* [8786](https://github.com/grafana/loki/pull/8786) **DylanGuedes**: Ingester: add new /ingester/prepare_shutdown endpoint.
+* [8744](https://github.com/grafana/loki/pull/8744) **dannykopping**: Ruler: remote rule evaluation.
+* [8670](https://github.com/grafana/loki/pull/8670) **salvacorts** Introduce two new limits to refuse log and metric queries that would read too much data.
+* [8918](https://github.com/grafana/loki/pull/8918) **salvacorts** Introduce limit to require at least a number label matchers on metric and log queries.
+* [8909](https://github.com/grafana/loki/pull/8909) **salvacorts** Requests to `/loki/api/v1/index/stats` are split in 24h intervals.
+* [8732](https://github.com/grafana/loki/pull/8732) **abaguas**: azure: respect retry config before cancelling the context
+* [9206](https://github.com/grafana/loki/pull/9206) **dannykopping**: Ruler: log rule evaluation detail.
+* [9184](https://github.com/grafana/loki/pull/9184) **periklis**: Bump dskit to introduce IPv6 support for memberlist
```
Comment (dannykopping) [inline CHANGELOG.md:26]: I think you just want to include the last item, no?

File: docs/sources/configuration/_index.md [hunk 2]
```diff
@@ -1218,6 +1222,11 @@ lifecycler:
   # CLI flag: -ingester.lifecycler.interface
   [interface_names: <list of strings> | default = [<private network interfaces>]]
 
+  # Enable IPv6 support. Required to make use of IP addresses from IPv6
+  # interfaces.
+  # CLI flag: -ingester.enable-inet6
+  [enable_inet6: <boolean> | default = false]
```
Comment (dannykopping) [inline docs/sources/configuration/_index.md:1228]: Is it possible to rename this to `enable_ipv6`?

Comment (periklis) [reply docs/sources/configuration/_index.md:1228]: Would mean to fix in main too, but sure we can do.

Comment (dannykopping) [reply docs/sources/configuration/_index.md:1228]: Can we not alias this field differently in the struct that embeds it?

**Review by dannykopping — APPROVED** [review]

**Pull request merged**